### PR TITLE
Show GroundPrimitive shadow volume

### PR DESCRIPTION
**Pull request by bagnell**
Adds a debug option to show the shadow volume of a `GroundPrimitive`.

CC @tfili

### CODE REVIEW

**Comment by pjcozzi** [comment]
Update CHANGES.md.

Add a test.

File: Source/Scene/GroundPrimitive.js [hunk 1]
```diff
@@ -205,6 +223,7 @@ define([
         this._readyPromise = when.defer();
 
         this._primitive = undefined;
+        this._debugPrimitive = undefined;
```
Comment (pjcozzi) [inline Source/Scene/GroundPrimitive.js]: Does this get destroyed when `destroy` is called or when `debugShowShadowVolume` is false?

Comment (bagnell) [inline Source/Scene/GroundPrimitive.js]: No, thanks for catching that.

**Comment by bagnell** [comment]
@pjcozzi This is ready for another look.

**Pull request merged**